api/user/schema.py

```diff
@@ -159,6 +159,14 @@ def mutate(self, info, email, **kwargs):
         return DeleteUser(user=exact_query_user)
 
 
+def restrictingAdmin(new_role):
```

Comment:
I think this function might be better placed in a helpers folder.

Reply:
done